### ISSUE TITLE
Missing a stage 2 loadshedding in Cape Town Area 7 + 15 today

### ISSUE THREAD

**Issue by afrith** [beyarkay/eskom-calendar#99]
```
$ curl -sL https://github.com/beyarkay/eskom-calendar/releases/download/latest/machine_friendly.csv | grep city-of-cape-town-area-7 | grep 2022-09-30
city-of-cape-town-area-7,2022-09-30T00:00:00+02:00,2022-09-30T00:30:00+02:00,4,"https://twitter.com/CityofCT/status/1575485965983895553"
city-of-cape-town-area-7,2022-09-30T22:00:00+02:00,2022-10-01T00:30:00+02:00,4,"https://twitter.com/CityofCT/status/1575485965983895553"

$ curl -sL https://github.com/beyarkay/eskom-calendar/releases/download/latest/machine_friendly.csv | grep city-of-cape-town-area-15 | grep 2022-09-30
city-of-cape-town-area-15,2022-09-30T00:00:00+02:00,2022-09-30T00:30:00+02:00,4,"https://twitter.com/CityofCT/status/1575485965983895553"
city-of-cape-town-area-15,2022-09-30T22:00:00+02:00,2022-10-01T00:30:00+02:00,4,"https://twitter.com/CityofCT/status/1575485965983895553"
```

`manually_specified.yaml` correctly states stage 2 for COCT from 06:00 to 22:00 - which means loadshedding at 14:00 for areas 7 and 15, but neither show up in the calendars.

**Comment by beyarkay** [comment]
Hey! thanks for raising the bug. I should be able to fix it this weekend. It's an issue in how `src/parse_coct_html.py` parses the HTML source for the loadshedding, but I'm not entirely sure why it's not getting the entire thing.

**Comment by afrith** [comment]
Thanks! Not sure if this helps but it might not actually be necessary to parse a source at all. The Cape Town loadshedding schedule is algorithmic - at [confluence/loadshedding](https://github.com/confluence/loadshedding) she's worked out the calculations that determine the schedule.

**Comment by beyarkay** [comment]
Ahh thats super cool, I didn't know about [confluence/loadshedding](https://github.com/confluence/loadshedding)!

I'll check it out, but I did some more digging and I think it'll actually be a bit easier to fix than I first thought, it's related to a duplication bug that I fixed in 5cb1d5bdd49bc70061653fae225b14ac3cb607db but apparently I also messed something up.

**Comment by beyarkay** [comment]
Should be fixed by [9cc597b](https://github.com/beyarkay/eskom-calendar/commit/9cc597b5e66fef3021ec423ecb26c5c555dfcf7f), we'll see if the [build and tests](https://github.com/beyarkay/eskom-calendar/actions/runs/3158163346) succeed and then I'll close the issue.

**Comment by beyarkay** [comment]
Should be fixed now:
```
$ curl -sL https://github.com/beyarkay/eskom-calendar/releases/download/latest/machine_friendly.csv | grep city-of-cape-town-area-7 | grep 2022-09-30
city-of-cape-town-area-7,2022-09-30T00:00:00+02:00,2022-09-30T00:30:00+02:00,4,"https://twitter.com/CityofCT/status/1575485965983895553"
city-of-cape-town-area-7,2022-09-30T14:00:00+02:00,2022-09-30T16:30:00+02:00,2,"https://twitter.com/CityofCT/status/1575485965983895553"
city-of-cape-town-area-7,2022-09-30T22:00:00+02:00,2022-10-01T00:30:00+02:00,4,"https://twitter.com/CityofCT/status/1575485965983895553"
```
```
$ curl -sL https://github.com/beyarkay/eskom-calendar/releases/download/latest/machine_friendly.csv | grep city-of-cape-town-area-15 | grep 2022-09-30
city-of-cape-town-area-15,2022-09-30T00:00:00+02:00,2022-09-30T00:30:00+02:00,4,"https://twitter.com/CityofCT/status/1575485965983895553"
city-of-cape-town-area-15,2022-09-30T14:00:00+02:00,2022-09-30T16:30:00+02:00,2,"https://twitter.com/CityofCT/status/1575485965983895553"
city-of-cape-town-area-15,2022-09-30T22:00:00+02:00,2022-10-01T00:30:00+02:00,4,"https://twitter.com/CityofCT/status/1575485965983895553"
```

Make sure your calendar app reloads the calendar subscription and you should be good to go! Google calendar sometimes takes up to 24 hours to reload, so it might be easier to unsubscribe/resubscribe if you want the changes immediately.
